arduino/libraries/librariesindex/index.go

```diff
@@ -136,6 +137,10 @@ func (idx *Index) FindLibraryUpdate(lib *libraries.Library) *Release {
 	if indexLib == nil {
 		return nil
 	}
+	if lib.Version == nil {
+		fmt.Printf("[WARN] version for library loaded from %s is nil\n", lib.InstallDir)
```

Comment:
Remove this line please, printing in this package might cause issues.

Reply:
Done.